src/ImageSharp.Drawing/project.json

```diff
@@ -0,0 +1,82 @@
+{
+  "version": "1.0.0-alpha1-*",
+  "title": "ImageSharp.Drawing",
+  "description": "A cross-platform library for the processing of image files; written in C#",
+  "authors": [
+    "James Jackson-South and contributors"
+  ],
+  "packOptions": {
+    "owners": [
+      "James Jackson-South and contributors"
+    ],
+    "projectUrl": "https://github.com/JimBobSquarePants/ImageSharp",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "iconUrl": "https://raw.githubusercontent.com/JimBobSquarePants/ImageSharp/master/build/icons/imagesharp-logo-128.png",
+    "requireLicenseAcceptance": false,
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/JimBobSquarePants/ImageSharp"
+    },
+    "tags": [
+      "Image Resize Crop Gif Jpg Jpeg Bitmap Png Core"
```

Comment:
We should probably have a look at the tags for each project, we could do that later though. Same goes for the description.